project_resource_calendar/report/calendar_event_report_view.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <report id="project_resource_calendar_report"
```

Comment:
project_resource_calendar_weekly_event_report

Reply:
this is not a view better to rename the file to (calendar_event_weekly_report.xml)